constants.yml

```diff
@@ -43,6 +44,7 @@ company_type:
     'private-unlimited' : "Private unlimited company"
     'ltd' : "Private limited company"
     'plc' : "Public limited company"
+    'pcc' : "Protected cell company"
```

Comment:
Should this be `protected-cell-company` to be consistent with **most** of the rest of this list?